files/build_templates/per_namespace/syncd.service.j2

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=syncd service
+Requires=database{% if multi_instance == 'true' %}@%i{% endif %}.service
+After=database{% if multi_instance == 'true' %}@%i{% endif %}.service
+After=swss{% if multi_instance == 'true' %}@%i{% endif %}.service
+{% if multi_instance == 'true' and sonic_asic_platform == 'vs' %}
+Requires=topology.service
+After=topology.service
+{% endif %}
+{% if sonic_asic_platform == 'broadcom' %}
+Requires=opennsl-modules.service
+{% elif sonic_asic_platform == 'nephos' %}
+Requires=nps-modules-4.9.0-11-2-amd64.service
+{% endif %}
+{% if sonic_asic_platform == 'broadcom' %}
+After=opennsl-modules.service
+{% elif sonic_asic_platform == 'nephos' %}
+After=nps-modules-4.9.0-11-2-amd64.service
+{% endif %}
```

Comment:
Since we're now grouping by service name instead of by "Before=", Requires=", "After=", etc., this can be simplified, also, as follows:

```
{% if sonic_asic_platform == 'broadcom' %}
Requires=opennsl-modules.service
After=opennsl-modules.service
{% elif sonic_asic_platform == 'nephos' %}
Requires=nps-modules-4.9.0-11-2-amd64.service
After=nps-modules-4.9.0-11-2-amd64.service
{% endif %}
```

Reply:
updated as per comment.